### PR TITLE
Update Terraform Feature

### PR DESCRIPTION
**Pull request by bamurtaugh**
Update based on https://github.com/microsoft/vscode-dev-containers/issues/1608.

### CODE REVIEW

**Review by samruddhikhandale — APPROVED** [review]
Thanks for making this quick change! ✨

File: src/terraform/devcontainer-feature.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
             "settings": {
                 "terraform.languageServer": {
                     "enabled": true,
-                    "args": []
+                    "args": ["serve"]
```
Comment (samruddhikhandale) [inline src/terraform/devcontainer-feature.json:46]:
```suggestion
                    "args": [
                        "serve"
                    ]
```

File: src/terraform/devcontainer-feature.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
             "settings": {
                 "terraform.languageServer": {
```
Comment (samruddhikhandale) [inline src/terraform/devcontainer-feature.json:44]: Also bump the patch version here https://github.com/devcontainers/features/blob/5336b419313fd75b01da75766e5b1fc1ddefb2c9/src/terraform/devcontainer-feature.json#L3 ? We release features every time a PR is merged, these changes won't take effect unless there is a version change from what currently exists :)

Comment (bamurtaugh) [reply src/terraform/devcontainer-feature.json:44]: Thanks for catching! Just bumped to 1.0.4, let me know if it looks alright to you 😄

**Review by samruddhikhandale — APPROVED** [review]
Thank you!

File: src/terraform/devcontainer-feature.json (hunk 2 above)
Comment (samruddhikhandale) [reply src/terraform/devcontainer-feature.json:44]: Looks good! 👍

**Pull request merged**